src/client/opamListCommand.ml

```diff
@@ -743,8 +743,13 @@ let info st ~fields ~raw_opam ~where ?normalise ?(show_empty=false) atoms =
           else acc)
         [] (List.rev fields)
     in
+    let cut =
```

Comment:
I believe you want `cut = None | Some `Truncate`, then `print_table ?cut`, to preserve the default wrapping when on a terminal

Reply:
updated